entrypoint.sh

```diff
@@ -100,4 +100,6 @@ for filter in ${filters}; do
   ln -sf "/data/filter.d/${filter}" "/etc/fail2ban/filter.d/"
 done
 
+tail -f ${F2B_LOG_TARGET} &
```

Comment:
That doesn't look right, `F2B_LOG_TARGET` can also be `STDOUT`, `STDERR` or `SYSLOG` which will not work here.